readability

Whennya opened the pull request:
This branch was made prior to finishing of https://github.com/notkoen/Music-Names/pull/1/ , ***SHOULD*** contain all changes within the last bit of there, currently this branch uses an arbitrary amount of tabs + spaces based on the longest length file name so all songs are the same distance so its less difficult to read, may be issues but want to make before i go degen raids this week so isn't sitting on backburner, surely didn't screw anything up

Whennya committed:
import from work repo

Add
- Bunch of cfgs
Fix
- Offliner misformat

Whennya committed:
add

Add
- ze_system_v1
- ze_minecraft_universe_v1_5s
- ze_minecraft_journey_v1_test_2
- ze_minecraft_adventure_v1_2c

Whennya committed:
add

Add
- ze_terraria_v3_2
- ze_luciddreams_v3_6_f2
- ze_light_shadow_l1_4
- ze_immortal_flame_v1_2
- ze_frozentemple_b8_2
- ze_flower_a1_s4
Fix
- ze_system_v1

Whennya committed:
Update ze_666_crazy_escape_v2_css2f.cfg

Whennya committed:
add

Add
- ze_conflict_v4_s2
- ze_ffxii_ridorana_cataract_b6
- ze_otakuroom_v5_6fs
- ze_serpentis_temple_v2a
Incomplete
- ze_luigismanison_fix2

Whennya committed:
add

- ze_2049_css1
- ze_ffxii_mt_bur_omisace_v6_beta6fix
- ze_randomizer_c1s4

Whennya committed:
Update ze_666_crazy_escape_v2_css2f.cfg

Whennya committed:
fix formatting on some imported configs

Whennya committed:
Update ze_m0w0m_b2.cfg

Whennya committed:
start

Whennya committed:
tab out a bunch of files

Whennya committed:
import changes

import final changes from last commit in
https://github.com/notkoen/Music-Names/pull/1/

Whennya committed:
finish

fixes some files with missing extensions / caps in file name

Whennya committed:
Update ze_biohazard_manor_v4a_004.cfg

Whennya committed:
Merge branch 'main' into consistent-formatting

Whennya committed:
fix merge conflict adding new lines to files

notkoen commented on musicname/ze_biohazard3_nemesis_b5_2s.cfg:
Remove white space and use spaces instead of tabs

notkoen commented on musicname/ze_deadcore_s7.cfg:
- `(Silent Spica)` seems to be extra information for `music_b_escape.mp3`
- Typo with `Instrumental` for `music_d_ex_escape.mp3`
- Capitalize and change `(main theme menu)` to `(Main Menu Theme)`
- I think `An (AcuticNotes)` should just be `AcuticNotes` for brevity

notkoen commented on musicname/ze_ffvii_cosmo_canyon_v5fix.cfg:
If possible, can the names of music be matched with `ze_ffvii_cosmo_canyon_v2_1` config? The formatting for `Final Fantasy VII (FF7) OST` can be changed to just `Final Fantasy VII OST`

notkoen commented on musicname/ze_ffvii_mako_reactor_v6_b08.cfg:
I was going to change the music name to `Final Fantasy VII OST` or just `FFVII OST` and `FFVII AC` to `FFVII: Advent Children` but you can decide if you want to change all these up.
Also `6zm_c_kh2_fate_of_the_unknown_mp3` does not have file extension. I would presume it's supposed to be `6zm_c_kh2_fate_of_the_unknown.mp3`

notkoen commented on musicname/ze_ffvii_malgo_reactor_css4_2.cfg:
Capital music names

notkoen commented on musicname/ze_ffxii_paramina_rift_v1_4.cfg:
Same thing here, calling it `Final Fantasy XII OST` for more clarity

notkoen commented on musicname/ze_ffxii_westersand_v7_2.cfg:
Same thing here with `Final Fantasy XII OST`

notkoen commented on musicname/ze_ffxii_westersand_v8zeta1.cfg:
Same thing as above

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg:
Some of the song names do not follow `<Artist> - <Song Title>` format

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v6.cfg:
Some music doesn't follow `<Artist> - <Song Title>` format

notkoen commented on musicname/ze_grau_s3_2.cfg:
Capital music names

notkoen commented on musicname/ze_gris_fyk.cfg:
Capital music names

notkoen commented on musicname/ze_shroomforest3_b6_2.cfg:
Nitpick: Add `OST` to the end of games

Whennya replied:
Should these follow the < artist - song > format? this is using the one currently live and some of them are setup like
< Source - Artist - Song > 
```    "doom - mick gordon - bfg division.mp3"       "Doom - Mick Gordon - BFG Division"```
in general i think this one has some weird formatting ill go back and fix later after i go through and fix the rest

notkoen replied:
Mick Gordon - BFG Division (Doom OST)

notkoen commented on musicname/ze_skill_escape_v09s.cfg:
Capital file names

notkoen commented on musicname/ze_slender_escape_b4.cfg:
Would prefer `Dark Knight OST`

Whennya replied:
Alternatively can be:
"Hans Zimmer - A Watchful Guardian"
"Hans Zimmer - Why Do We Fall?" 
which are the actual track links instead of the movies OST, lmk which one would be better, for now ill just update to the entire title + OST

notkoen replied:
Could just add `(Dark Knight OST)` to the end

notkoen commented on musicname/ze_stilshrine_of_miriam_v7_2s.cfg:
Inconsistent formatting with tabs and spaces

notkoen commented on musicname/ze_surf_vortex_v1_9s.cfg:
Trim last line

notkoen commented on musicname/ze_totally_new_wester.cfg:
Add `OST` to the end of `Final Fantasy XII`

notkoen commented on musicname/ze_uchiha_legacy_v1_3f.cfg:
Inconsistent formatting, trim empty line

notkoen commented on musicname/ze_valkvang_s1.cfg:
Trim whitespace

notkoen commented on musicname/ze_visualizer_css2.cfg:
Nitpick: `DANGER - 4h30` should match the others and be `Danger - 4h30` imo

Whennya committed:
fixex

Hopefully fix all formatting / consistency issues between file names + tabs / space combinations

Whennya committed:
fix ze_ffvii_mako_reactor_v6_b08.cfg song names

notkoen commented on musicname/ze_ffvii_cosmo_canyon_v2_1.cfg:
For better consistency, I would hope `[FFVII OST]` on line 3 is changed to `Final Fantasy VII OST`, as well `FFXIII OST` on line 12 to `Final Fantasy XII OST`

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg:
```diff
@@ -1,16 +1,16 @@
+    "savage_copied_city.mp3"                    "NieR Automata OST - Copied City (Sequential Mix) v2"
+    "normal_wanderes_palace_theme.mp3"          "FFXIV OST - Wanderer's Palace HM Theme"
+    "normal_the_praetorium_theme.mp3"           "FFXIV OST - The Praetorium Theme (Penitus)"
+    "normal_a_fine_death.mp3"                   "FFXIV OST - A Fine Death (mid-Dungeon Boss Theme)"
```
`(mid-Dungeon Boss Theme)` should have capital m -> `(Mid-Dungeon Boss Theme)
```suggestion
    "normal_a_fine_death.mp3"                   "FFXIV OST - A Fine Death (Mid-Dungeon Boss Theme)"
```

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg:
```diff
@@ -1,16 +1,16 @@
+    "ultimate_starringchild.mp3"                "Aimer - StarRingChild"
+    "ultimate_ref_rain.mp3"                     "Aimer - Ref:rain"
+    "ultimate_re_i_am.mp3"                      "Aimer - Re I Am"
+    "savage_unseen_abyss.mp3"                   "FFXIII-2 OST - Jet Bahamut (Unseen Abyss)"
```
This is something I'm not sure what to do. Should `Final Fantasy` be abbreviated as `FF` as is? These song names use `FFXIV OST` but some other configs do `Final Fantasy XIV OST`. Which one do you prefer? I prefer the latter, although it'll require you to go back and update all the configs.

Whennya committed:
Update musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg

Co-authored-by: Aaron <45914779+notkoen@users.noreply.github.com>

Whennya committed:
Consolidate Final Fantasy -> FF

Whennya committed:
Update ze_ffvii_cosmo_canyon_v5fix.cfg

Whennya committed:
forgot this

notkoen commented on musicname/ze_ffvii_cosmo_canyon_v2_1.cfg:
```diff
@@ -1,24 +1,24 @@
+    "ffvii_hurry_up.mp3"                        "FFVII OST - Hurry Faster"
+    "ffvii_granitev2.mp3"                       "Pendulum - Granite"
+    "ffvii_hurry.mp3"                           "FFVII OST - Hurry"
+    "ffvii_conflict.mp3"                        "Conflict - Takeharu Ishimoto"
```
Fix music name format to `<Artist> - <Music Name>`

notkoen commented on musicname/ze_ffvii_cosmo_canyon_v5fix.cfg:
```diff
@@ -1,22 +1,22 @@
+    "ffvii_hurry_up.mp3"                    "FFVII OST - Hurry Faster"
+    "ffvii_granitev2.mp3"                   "Pendulum - Granite"
+    "ffvii_hurry.mp3"                       "FFVII OST - Hurry"
+    "ffvii_conflict.mp3"                    "Conflict - Takeharu Ishimoto"
```
Same as above

Whennya committed:
fixes
